src/components/CurationPage/CurationPage.tsx

```diff
@@ -172,13 +172,15 @@ export default class CurationPage extends React.PureComponent<Props, State> {
             </Table>
           </Section>
           {totalPages > 1 && (
-            <Pagination
-              firstItem={null}
-              lastItem={null}
-              totalPages={totalPages}
-              activePage={page}
-              onPageChange={(_event, props) => this.setState({ page: +props.activePage! })}
-            />
+            <div className="pagination">
```

Comment:
Would you mind passing the pagination class as a prop to `Pagination` (in `className`) instead of creating a new div?

Reply:
`Pagination` does not have a className prop afaik. Will check